src/ray/util/process.cc

```diff
@@ -299,32 +280,19 @@ ProcessFD &ProcessFD::operator=(ProcessFD &&other) {
   return *this;
 }
 
-intptr_t ProcessFD::CloneFD() const {
```

Comment:
it is not used now btw? (I assume yes given deleting copy constructor just succeeds?)

Reply:
it's not used anywhere